src/api/controllers/articleController.js

```diff
@@ -53,6 +52,110 @@ export default class ArticleController {
     }
   }
 
+  /**
+   * publish a drafted article
+   * @static
+   * @param {Object} req the request
+   * @param {Object} res the response to be sent
+   * @memberof ArticleController
+   * @returns {Object} res
+   */
+  static async publish(req, res) {
+    const { slug } = req.params;
+    const article = await Article.update({
+      status: 'published'
+    },
+    {
```

Comment:
@alainburindi I know that all these attributes have a `not Null ` constraint.
Why would you want to check this yet an article will never be stored if they are null ?

Reply:
so that an article can not be published, if there is an empty field